fe/src/components/Sale/SaleHeader/index.tsx

```diff
@@ -5,7 +5,11 @@ import PATH from '@constants/routerPath';
 import Navbar from '@components/molecules/Navbar';
 import Button from '@atoms/Buttons/Button';
 
-const SaleHeader = () => {
+interface SaleHeaderProps {
+  handleSubmit: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
```

Comment:
`MouseEvent`가 중복되는 것 같은데, 아래와 같이 표현하는 건 어떨까요?

```suggestion
  handleSubmit: (event: React.MouseEvent<HTMLButtonElement>) => void;
```